packages/exchange-ui-core/src/containers/Exchange.tsx

```diff
@@ -8,7 +8,8 @@ import {
     useIntervalFetch,
     getCountry,
     getUserOffchain,
-    setLoading
+    setLoading,
+    getEnvironment
```

Comment:
In exchange-ui-core we should use getEnvironment from the exchange-ui-core package, route: "../features/general".
Maybe it could be renamed to getExchangeEnvironment to avoid misunderstanding.

Reply:
fixed, thank you